legacy/firmware/ethereum_onekey.c

```
@@ -874,6 +1240,437 @@ static bool ethereum_signing_handle_nft(struct signing_params *params,
+    memcpy(to, data, 20);
+    char *to_str = decode_typed_data(to, 20, "address");
+    display_info_set_value(&display_info, to_str);
+    free(to_str);
+    display_info_add_item_name(&display_info, "value", 0);
+    char *value_str = decode_typed_data(data + 20, 32, "uint");
+    display_info_set_value(&display_info, value_str);
+    free(value_str);
+    // display_info_add_item_name(&display_info, "operation", 0);
+    uint8_t operation = data[105];
+    char operation_str[16];
+    if (operation == 0) {
```

Clang (14.0.6)

[note] +2, including nesting penalty of 1, nesting level increased to 2

(clang)